<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="dev.maurer.ServiceTests.EmployeeServiceTests" tests="6" skipped="0" failures="0" errors="0" timestamp="2021-03-17T23:19:47" hostname="DESKTOP-RUU1C4G" time="0.048">
  <properties/>
  <testcase name="employeeLoginExceptionTest()" classname="dev.maurer.ServiceTests.EmployeeServiceTests" time="0.04"/>
  <testcase name="getEmployeeByIdExceptionTest()" classname="dev.maurer.ServiceTests.EmployeeServiceTests" time="0.002"/>
  <testcase name="getManagerByIdTest()" classname="dev.maurer.ServiceTests.EmployeeServiceTests" time="0.001"/>
  <testcase name="employeeServiceEmployeeLoginTest()" classname="dev.maurer.ServiceTests.EmployeeServiceTests" time="0.001"/>
  <testcase name="employeeServiceManagerLoginTest()" classname="dev.maurer.ServiceTests.EmployeeServiceTests" time="0.001"/>
  <testcase name="getEmployeeByIdTest()" classname="dev.maurer.ServiceTests.EmployeeServiceTests" time="0.001"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
